monitoring/Makefile

```diff
@@ -17,7 +17,7 @@ push: build
 
 # FIXME use stateful set or prometheus, grafana
 deploy: push
-	kubectl -n monitoring delete deployments prometheus grafana
+	kubectl -n monitoring delete deployments prometheus
```

Comment:
We need to scale down grafana if it exists before redeploying right?

Reply:
If it already exists and we use `kubectl apply` my understanding is that it will either remain unchanged or redeploy itself if there are updates to the monitoring.yaml file. I don't think we need to explicitly kill it?